types/tsconfig.json

```diff
@@ -8,9 +8,6 @@
     "noUnusedParameters": true,
     "noFallthroughCasesInSwitch": true,
     "noImplicitReturns": true,
-    "noImplicitAny": true,
```

Comment:
Newer version of TypeScript throws an error if these are used together with `strict`, which supersedes them.